extensions/wikia/AdEngine/js/spec/pageFairDetection.spec.js

```
@@ -80,18 +80,22 @@ describe('Module ext.wikia.adEngine.pageFairDetection', function () {
 		expect(mocks.window.ads.runtime.pf.blocking).toBe(false);
 	});
 
+	it('should throw an exception if its try to initialize without correct context', function () {
```

`should throw an exception if its tries to initialize without correct context`